### PR TITLE
Updated OAuth Facebook login, screenshot, code

### PR DESCRIPTION
**Pull request by wadepickett**
Updated Facebook login instructions for 3.0 and for latest FaceBook Developer registration / App UI:

[Internal review](https://review.docs.microsoft.com/en-us/aspnet/core/security/authentication/social/facebook-logins?view=aspnetcore-3.0&branch=pr-en-us-15838)

- Updated facebook-logins.md
- Updated screenshots
- Updated the include: default-settings.md 
- Updated code example to account for 3.0 changes where we no longer need to call services.AddDefaultIdentity and Bootstrap4 is now already default.
- Removed instruction specific to versions prior to 3.0.

### CODE REVIEW

**Comment by wadepickett** [comment]
Sorry, not ready for review yet!  Set it as draft but then accidentally hit "ready for review" as I was trying to click on the build verification links while the page was adjusting itself.

**Comment by wadepickett** [comment]
@Rick-Anderson, after separating out version specific instructions in this FB OAuth topic, I just remembered you had decided to not provide previous version instruction (prior to 3.0) for the overview topic this FB topic takes a dependency on.   Should I only include 3.0 content in this FB topic and delete previous version targeted instruction?  I assume so, but thought I better check.

**Comment by Rick-Anderson** [comment]
Yes, just do the latest version. This stuff changes too frequently.

File: aspnetcore/security/authentication/social/facebook-logins.md [hunk 1]
```diff
@@ -4,32 +4,36 @@ author: rick-anderson
...
 
 By [Valeriy Novytskyy](https://github.com/01binary) and [Rick Anderson](https://twitter.com/RickAndMSFT)
 
-This tutorial with code examples shows how to enable your users to sign in with their Facebook account using a sample ASP.NET Core 2.0 project created on the [previous page](xref:security/authentication/social/index). We start by creating a Facebook App ID by following the [official steps](https://developers.facebook.com).
+This tutorial with code examples shows how to enable your users to sign in with their Facebook account using a sample ASP.NET Core 3.0 project created on the [previous page](xref:security/authentication/social/index). We start by creating a Facebook App ID by following the [official steps](https://developers.facebook.com).
 
 ## Create the app in Facebook
 
-* Navigate to the [Facebook Developers app](https://developers.facebook.com/apps/) page and sign in. If you don't already have a Facebook account, use the **Sign up for Facebook** link on the login page to create one.
+* Install [Microsoft.AspNetCore.Authentication.Facebook](https://www.nuget.org/packages/Microsoft.AspNetCore.Authentication.Facebook).
```
Comment (Rick-Anderson) [inline aspnetcore/security/authentication/social/facebook-logins.md]: * Add the [Microsoft.AspNetCore.Authentication.Facebook](https://www.nuget.org/packages/Microsoft.AspNetCore.Authentication.Facebook) NuGet package to the project.

File: aspnetcore/security/authentication/social/facebook-logins.md [hunk 2]
```diff
@@ -4,32 +4,36 @@ author: rick-anderson
...
+* From the **My Apps** menu select **Create App** to create a new App ID.
 
    ![Facebook for developers portal open in Microsoft Edge](index/_static/FBMyApps.png)
 
 * Fill out the form and tap the **Create App ID** button.
 
   ![Create a New App ID form](index/_static/FBNewAppId.png)
 
-* On the **Select a product** page, click **Set Up** on the **Facebook Login** card.
+* On your new App card, select **Add a Product**.  On the **Facebook Login** card, click **Set Up**
```
Comment (Rick-Anderson) [inline aspnetcore/security/authentication/social/facebook-logins.md]: * On the new App card, select **Add a Product**.  On the **Facebook Login** card, click **Set Up**

Comment (Rick-Anderson) [reply aspnetcore/security/authentication/social/facebook-logins.md]: Avoid you, your expect when it adds clarity. Sometimes it sounds better with you/your. In this case, it's better without it.
If you agree with the suggestions batch add them from the files tab.

File: aspnetcore/security/authentication/social/facebook-logins.md [hunk 3]
```diff
@@ -4,32 +4,36 @@ author: rick-anderson
...
 
   ![Create a New App ID form](index/_static/FBNewAppId.png)
 
-* On the **Select a product** page, click **Set Up** on the **Facebook Login** card.
+* On your new App card, select **Add a Product**.  On the **Facebook Login** card, click **Set Up** 
 
   ![Product Setup page](index/_static/FBProductSetup.png)
 
-* The **Quickstart** wizard will launch with **Choose a Platform** as the first page. Bypass the wizard for now by clicking the **Settings** link in the menu on the left:
+* The **Quickstart** wizard will launch with **Choose a Platform** as the first page. Bypass the wizard for now by clicking the **FaceBook Login** **Settings** link in the menu on the lower left:
```
Comment (Rick-Anderson) [inline aspnetcore/security/authentication/social/facebook-logins.md]: * The **Quickstart** wizard launches with **Choose a Platform** as the first page. Bypass the wizard for now by clicking the **FaceBook Login** **Settings** link in the menu on the lower left:

Comment (Rick-Anderson) [reply aspnetcore/security/authentication/social/facebook-logins.md]: Per style guide and acrolynx, avoid FT.

File: aspnetcore/security/authentication/social/facebook-logins.md [hunk 4]
```diff
@@ -142,7 +117,7 @@ You are now logged in using your Facebook credentials:
 
 ## Next steps
 
-* Add the [Microsoft.AspNetCore.Authentication.Facebook](https://www.nuget.org/packages/Microsoft.AspNetCore.Authentication.Facebook) NuGet package to your project for advanced Facebook authentication scenarios. This package isn't required to integrate Facebook external login functionality with your app. 
+* Add the [Microsoft.AspNetCore.Authentication.Facebook](https://www.nuget.org/packages/Microsoft.AspNetCore.Authentication.Facebook) NuGet package to your project for advanced Facebook authentication scenarios. This package isn't required to integrate Facebook external login functionality with your app.
```
Comment (Rick-Anderson) [inline aspnetcore/security/authentication/social/facebook-logins.md]: Delete this line. You have them add it at the start of the tutorial. It used to be included but no longer is.

**Comment by wadepickett** [comment]
@Rick-Anderson , all requested changes from your review now in.  Thanks!

**Pull request merged**